html/user-list.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <title>User List</title>
     <meta content="text/html; charset=UTF-8">
+    <link rel="stylesheet" type="text/css" href="../css/style.css" />
 </head>
 <body>
```

Comment:
The user cards don't have fixed width, so the "Jordan Smith" card is slightly wider than the others.